### ISSUE TITLE
Use forked Repo until PR is merged at node-mihome

### ISSUE THREAD

**Issue by ylmzdev** [Pittini/iobroker-nodemihome#12]
you can use "Pittini/node-mihome" as required NPM module in iobroker until the PR is accepted/merged at node-mihome.

![image](https://user-images.githubusercontent.com/72867745/108407301-dbae7f00-7223-11eb-9754-f4be9d077eed.png)

adding the files this way will help you to avoid copying the files multiple times.

@Pittini maybe you can add this information to README :)

**Comment by Pittini** [comment]
Very nice, i did not know this works. Thank you. I'll add it to the readme next update.

**Comment by Wildbill-Z** [comment]
@ylmzdev I tested this yesterday and got some serious problems as described [HERE](https://forum.iobroker.net/post/586794) .
I was not able to update or install adapters and allways got some errors with "node-mohime-38". Thank Proxmox I was able to backup to the state before but unfortunately that killed the log also.

What is curious, I was not able to put "Pittini/node-mohome" in Javascript-Adapter and I had to put the whole path to Pitiini's Github repository/node-mihome.

Before restoring the backup I tried to delete the path in Javascript-adapter and deleted whole node-mihome in iobroker folder manually, but no way.

Greets, Jürgen
